bluebottle/activities/tests/test_api.py

```diff
@@ -626,8 +626,8 @@ def test_filter_segment(self):
         self.assertFacets(
             f'segment.{segment_type.slug}',
             {
-                f'segments.{matching_segment.pk}': len(matching),
-                f'segments.{other_segment.pk}': len(other)
+                f'{matching_segment.pk}': len(matching),
+                f'{other_segment.pk}': len(other)
```

Comment:
Still has to be cast to string though....
`f'{other_segment.pk}'` or `str(other_segment.pk)`
same difference :man_shrugging: